dev/snippets/config/templates/stateless_widget.tmpl

```diff
@@ -10,12 +10,13 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return new MaterialApp(
+    return MaterialApp(
       title: 'Flutter Code Sample for {{id}}',
-      theme: new ThemeData(
+      theme: ThemeData(
```

Comment:
Same comments as earlier about the theme and color parameters.

Reply:
Same fix. :-)